README.md

```diff
@@ -324,6 +324,96 @@ xsite:
       port: 7200
 ```
 
+Next the (local) port 7900 must be mapped to the port defined in the configuration (7200). On Docker for example by using `-p 7200:7900`
+
+Note: Caches won't be created automatically on the other side. So these must be created manually with the same name.
+
+So create a new cache either using the console (http://localhost:11222) or curl (or similar tool)
+`curl -XPOST -u <user>:<password> -H "Content-Type: application/xml" -d "@<path/to/cache.xml>" http://localhost:11222/rest/v2/caches/xsite-cache`
+  
+LON-Site:
+```
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsite-cache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC"/>
+      </backups>
+    </replicated-cache>
+  </cache-container>
+</infinispan>
+```
+
+NYC-Site:
+```
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsite-cache" statistics="true">
+      <backups>
+        <backup site="LON" strategy="SYNC"/>
+      </backups>
+    </replicated-cache>
+  </cache-container>
+</infinispan>
+```
+
+To test the replication, simply create a key on one site and search for it on the other site, where it will be available then, too.
+
+Docker command example for two sites on same system (but you have to use different ports for both sites):
+
+Config1:
+
+```
+jgroups:
+  diagnostics: true
+  encrypt: false
+xsite:
+  address: <ip of infinispan nyc>
+  name: NYC
+  port: 7300
+  backups:
+    - address: <ip of infinispan lon>
+      name: LON
+      port: 7200
```

Comment:
Include NYC

Reply:
not required as well. the operator configuration requires all sites but the "image" only requires the remote.